lib/puppet/util/windows/file.rb

```diff
@@ -89,10 +89,31 @@ def symlink(target, symlink)
   end
   module_function :symlink
 
+
+  def file_exist(file_name)
+    path = file_name.to_s
```

Comment:
I think you've lost the ability to support `WatchedFile`

Also, please keep the note about why we `to_s` here.  We might want to backfill tests for that if nothing broke during testing.